src/daemon/rpmostreed-transaction.c

```diff
@@ -490,6 +497,24 @@ transaction_constructed (GObject *object)
     }
 }
 
+static gboolean
+foreach_close_peer (gpointer key,
+                    gpointer value,
+                    gpointer user_data)
+{
+  GDBusConnection *conn = key;
+  g_dbus_connection_close_sync (conn, NULL, NULL);
+  return TRUE;
+}
+
+void
+rpmostreed_transaction_force_close (RpmostreedTransaction *transaction)
+{
+  RpmostreedTransactionPrivate *priv = rpmostreed_transaction_get_private (transaction);
```

Comment:
Should we sanity check here that `active == FALSE`? (And `g_debug` or something if that's not the case?)